src/main/java/core/basesyntax/IRandomSide.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface IRandomSide {

```

Comment:
you don't need this interface, remove it

Reply:
Done